concordium-node/src/grpc2.rs

```diff
@@ -880,6 +883,10 @@ pub mod server {
         net::SocketAddr,
         sync::{Arc, Mutex},
     };
+    use tokio::{
+        time,
+        time::{Duration, Instant},
```

Comment:
I found the use of this confusing since there is also std::time. Can you use qualified names instead of importing `time` here?